ccdImports/configFiles/CCD_Probate_Backoffice/CaseField.json

```diff
@@ -146,7 +146,7 @@
-  {"LiveFrom": "01/01/2017", "CaseTypeID": "GrantOfRepresentation", "ID": "solsStartPage", "Label": "## Submit grant of probate applications online\n\nYou must submit grant of probate applications online, unless an exemption applies.\n\nYou can submit applications for letters of administration (intestacy) and letters of administration with will annexed (admon will) online when certain conditions apply.\n\n<a href=\"https://www.gov.uk/guidance/probate-paper-applications-for-legal-professionals\" target=\"_blank\">Guidance on exemptions, conditions and when application must be submitted by paper</a>\n\nStep by step guidance on submitting an application online can be found at <a href=\"https://www.gov.uk/government/publications/myhmcts-how-to-apply-for-probate-online/apply-for-probate-with-myhmcts\" target=\"_blank\">Apply for probate with MyHMCTS</a>\n\n## If inheritance tax needs to be paid\nIf the estate has been valued and inheritance tax needs to be paid:\n* send the IHT400 and IHT421 form to HMRC\n* wait 20 working days before applying for the grant\n\nIf you do not wait 20 working days to submit the probate application, we cannot verify the IHT421 details from HMRC. This will delay the grant being issued.\n\n", "FieldType": "Label", "SecurityClassification": "Public"},
+    {"LiveFrom": "01/01/2017", "CaseTypeID": "GrantOfRepresentation", "ID": "solsStartPage", "Label": "## Submit grant of probate applications online\n\nYou must submit grant of probate applications online, unless an exemption applies.\n\nYou can submit applications for letters of administration (intestacy) and letters of administration with will annexed (admon will) online when certain conditions apply.\n\n<a href=\"https://www.gov.uk/guidance/probate-paper-applications-for-legal-professionals\" target=\"_blank\">Guidance on exemptions, conditions and when application must be submitted by paper</a>\n\nStep by step guidance on submitting an application online can be found at <a href=\"https://www.gov.uk/government/publications/myhmcts-how-to-apply-for-probate-online/apply-for-probate-with-myhmcts\" target=\"_blank\">Apply for probate with MyHMCTS</a>\n\n## If you have submitted IHT400 and IHT421 forms before {{ date }}\nYou must wait 20 working days after sending the IHT400 and IHT421 forms to HMRC before you apply for the grant.\n\nIf you do not wait, we will be unable to verify the IHT421 details from HMRC.This will delay the grant being issued.\n\n## If you have submitted an IHT400 form after{{ date }}\nYou must wait to receive a letter from HMRC with a probate code.You will need this code to apply.\n\nHMRC will send out the letter up to 20 working days after you submit the IHT400 form. If you have not received the letter,you should contact HMRC. \n\n", "FieldType": "Label", "SecurityClassification": "Public"},
```

Comment:
probably should have space after , in:
,you should contact HMRC
Apologies, didn't notice this first time

Reply:
I've put in the space